packages/next-dashboard/src/components/Statistic.js

```diff
@@ -1,21 +1,72 @@
 // @flow
-import React, { PureComponent } from 'react';
+import React, { type Node } from 'react';
+import classnames from 'classnames';
 
 import type { Statuses } from '../utils/types';
 
 export type Props = {
-  label?: React$Node,
-  value: React$Node,
-  prefix?: React$Node,
-  suffix?: React$Node,
-  periodValue?: React$Node,
-  periodText?: React$Node,
-  description?: React$Node,
+  label?: Node,
+  value: Node,
+  prefix?: Node,
+  suffix?: Node,
+  periodValue?: Node,
+  periodText?: Node,
+  description?: Node,
```

Comment:
```suggestion
  label?: ReactNode,
  value: ReactNode,
  prefix?: ReactNode,
  suffix?: ReactNode,
  periodValue?: ReactNode,
  periodText?: ReactNode,
  description?: ReactNode,
```

Reply:
If we're gonna adopt the pattern of importing the type from the react dependency, I suggest we do this.

Otherwise it conflicts with the [bulitin Node DOM class](https://developer.mozilla.org/en-US/docs/Web/API/Node)

Reply:
Hm, I mean I guess that's a fair point... maybe we could use `React$Node` for this one since it's so common, but otherwise try to stick to importing React types? or is that inconsistent...

Reply:
I feel we either do a pattern where we import types, or use flow builtin types otherwise it's inconistent. A lot of the imported types are just aliases for the builtin flow types anyway:

![image](https://user-images.githubusercontent.com/1394292/75783895-9cacc280-5d61-11ea-85b6-18157a041e95.png)

So I kinda feel the intended way is to just... use the React$[type] syntax everywhere?

Reply:
Also, the Node class is not just a theoretical possibility of conflict. Look at this file: https://github.com/PiJaAB/next-dashboard/blob/9369e7264aca30d3cc49529b372e7a7743fb191d/packages/next-dashboard/src/components/Modal.js#L47

Reply:
After further Slack discussion, we opted for [doing a namespace import][1] and then using `React.Node`. The reason for avoiding `React$Node` is to avoid relying on undocumented internal Flow types.

We'll see how this pans out with files also importing hooks; one option would bet o destructure them out of the imported namespace afterwards.

[1]: https://flow.org/en/docs/react/types/